### ISSUE TITLE
Time to execute CellularSpace:save()

### ISSUE THREAD

**Issue by pedro-andrade-inpe** [TerraME/terrame#1793]
It seems that the time to execute CellularSpace:save() was multiplied by three. Verify it.

**Comment by avancinirodrigo** [comment]
Review code associated to geometry in save().

**Comment by avancinirodrigo** [comment]
The time problem in save() is associated to the method that convert TerraLib::Geometry to OGR::Geometry.
It can be found in ogr::Transactor::add() method:
```cpp
          case te::dt::GEOMETRY_TYPE:
            {
              std::auto_ptr<te::gm::Geometry> geom(d->getGeometry(i));
              OGRGeometry* OGRgeom = Convert2OGR(geom.get());           //<<<< HERE
              feat->SetGeometryDirectly(OGRgeom);
            }
          break;
```


For the test, it was created a Profiling in Lua code that calculates the time in some specific places:
``` lua
t0 = os.clock()
-- some functions
print("SaveDataSet", string.format("elapsed time: %.2f\n", os.clock() - t0))
```

Using luccme package, it was simulated a model with a cellular space with 12214 cells that save 15 new attributes in a new cellular space at the end of the simulation.
The result time average to the simulation in add() method was 110 sec (shapefile) and 1.5 sec (postgis), **almost 10x faster**.
If it is removed the geometry for save to shapefile, the result is less than 1 sec. This proves the problem with geometry converter. 

My conclusion is, for big data or if you want performance, the best solution is work with postgis. 
Also, it is possible to create a parameter in cellular space that remove the geometry to save a new data, but the result cannot be used for new operations which use geometry.

**Comment by pedro-andrade-inpe** [comment]
Closed by #1884.
